### ISSUE TITLE
Disappearing unread counter

### ISSUE THREAD

**Issue by bmalkow** [mattermost/desktop#1429]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**
When I unlock my macOS, then unread messages marker disappearing, even if focus is not on Mattermost application.

**Environment**
- Operating System: macOS Catalina 10.15.7 (19H15)
- Mattermost Desktop App version: 4.6.0
- Mattermost Server version: 5.25.5

**Steps to reproduce**

1. Run Mattermost client
2. Be active on some chat
3. Switch active application to anything else, let Mattermost with this chat will be in the background
4. Lock computer by pressing control-command-q
5. wait for any incoming message in active chat
6. unlock computer
7. just after computer is unlocked, unread messages counter disappears

**Expected behavior**

Unread counter will be in place until I switch to Mattermost client again.

**Observed behavior**

I recorded screen when it happens
[Mattermost unread marker error.mov.zip](https://github.com/mattermost/desktop/files/5695536/Mattermost.unread.marker.error.mov.zip)

**Possible fixes**

**Comment by Willyfrog** [comment]
do you switch the app from the background to the foreground? this might cause the current channel to be marked as read.

**Comment by bmalkow** [comment]
No. I don't. A you can see on screencast, focus is on iTerm all the time.

**Comment by Willyfrog** [comment]
ah! I missed the screencast, sorry about that and thanks for providing it as it makes everything more clear.

I've tested 4.5.4 and doesn't show this behaviour so it is clearly a regression. I've created a [ticket to work on it](https://mattermost.atlassian.net/browse/MM-31429) feel free to add any details there.
